mule-apikit-module/src/main/java/org/mule/module/apikit/Router.java

```diff
@@ -128,7 +134,16 @@ private Throwable buildMessagingException(Throwable e) {
       return ApikitErrorTypes.throwErrorType((MuleRestException) e);
     }
 
-    return e;
+    if (e instanceof ComponentExecutionException) {
```

Comment:
When is this executed? `ComponentExecutionException` is only thrown by `ExecutableComponents`.

Reply:
this is executed when routing with non-blocking https://github.com/mulesoft/apikit/blob/7188c17b89597b6a1fa297edecf5fb3e63d00e86/mule-apikit-module/src/main/java/org/mule/module/apikit/Router.java#L125